test/util/ReduceCssCalcPrototype.spec.ts

```diff
@@ -0,0 +1,315 @@
+import reduceCSSCalc from 'reduce-css-calc';
```

Comment:
this file looks a lot cleaner, I like it more than the above. 

But still at the end of the day how much bundle size could this possibly save?

Reply:
I'll cleanup the commits, check bundle size and performance